### PR TITLE
Upgrade Strapi to v4.6.1

### LINKED ISSUE #1349: Upgrade Strapi to v4.6.1
Strapi v4.6.0 introduces improvements across the board, including a new import/export tool that is really helpful to improve the way we seed Govinor instances.

### PR DESCRIPTION
**Pull request by dhmacs**
closes #1349 

❗❗❗After this PR is merged Strapi should be redeployed in production ❗❗❗

cc @sterlinghirsh @danielbeardsley 

## QA

1. Login into the [strapi instance for this branch](https://upgrade-strapi-to-v4-6-0.govinor.com/admin)
2. Go to settings and check that the Strapi version is `v4.6.1`
3. Open [Vercel preview](https://react-commerce-git-upgrade-strapi-to-v4-6-0-ifixit.vercel.app/Parts)
4. Verify that product list page renders

### CODE REVIEW

**Comment by deltuh-vee** [comment]
QA 🎬 
* Strapi has been updated to version 4.6.1
* Product Page still renders

File: backend/tsconfig.json [hunk 1]
```diff
@@ -2,7 +2,9 @@
    "extends": "@strapi/typescript-utils/tsconfigs/server",
    "compilerOptions": {
       "outDir": "dist",
-      "rootDir": "."
+      "rootDir": ".",
+      "jsx": "react",
+      "lib": ["ES2020", "DOM"]
```
Comment (ardelato) [inline backend/tsconfig.json:7]: Why did we need to modify the `tsconfig` with this Strapi update?

**Review by ardelato — APPROVED** [review]
CR 👍 , but deploy_block ⚡ on question and confirmation from @danielbeardsley or @sterlinghirsh

File: backend/tsconfig.json (hunk 1 above)
Comment (dhmacs) [reply backend/tsconfig.json:7]: As weird as it sounds, when you're running certain Strapi commands, such as `strapi export`, they invoke a Typescript compilation, and that throws errors without these additional settings. 
I've also opened an issue on the Strapi repo to clarify this behavior https://github.com/strapi/strapi/issues/15707

**Comment by ardelato** [comment]
un_deploy_block 👍

**Comment by deltuh-vee** [comment]
Redeployed Strapi

**Pull request merged**